typespec-tests/src/main/java/com/_specs_/azure/core/lro/rpc/legacy/LegacyClient.java

```diff
@@ -172,7 +172,7 @@ public JobResult getJob(String jobId) {
      */
     @Generated
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public SyncPoller<JobResult, JobResult> beginCreateJob(JobData jobData) {
+    public SyncPoller<JobResult, Void> beginCreateJob(JobData jobData) {
```

Comment:
Does `Void` mean this `create` operation does not have final result? Wonder how to define in tsp if we want to have final result.

Reply:
Yes, if we follow the info given by LroMetadata, it is.

Reply:
I see. Thanks.